source/v2/greenlanes-openapi.yaml

```diff
@@ -0,0 +1,683 @@
+openapi: 3.1.0
+
+servers:
+  - url: https://www.trade-tariff.service.gov.uk/xi/api/v2/green_lanes
+
+security:
+- bearerAuth: []
+
+info:
+  version: "2"
+  title: Windsor Framework Green Lanes Categorisation API
+  description: |
+    <% content_for :head do %>
+      <link href="/green-lanes.xml" type="application/atom+xml" rel="alternate" title="Green Lanes api changes feed" />
+    <% end %>
+
+    The Green Lanes Categorisation API's provide access to categorisation data for the different Commodities and Subheadings within the Tariff.
```

Comment:
I might be wrong, do we need a the single quote here:  "API's provide" ?
or maybe `APIs provide` 
not sure

Reply:
Works fine - its just a YAML section?

![image](https://github.com/trade-tariff/trade-tariff-api-docs/assets/10818/618b85cb-83aa-4dc4-a4c4-9c7ad9a445a0)